### PR TITLE
Method to convert WKT to polygon

### PR DESCRIPTION
**Pull request by mateusppereira**
WKT is a very usefull way of write coords (https://en.wikipedia.org/wiki/Well-known_text_representation_of_geometry#Geometric_Objects)

### CODE REVIEW

**Comment by manuelbieh** [comment]
Great work, love it! Would be cool to also have support for `POINT (30 10.54321)` in `getLatitude` and `getLongitude` but with the current implementation it is not as easy as it seems to implement.

**Comment by manuelbieh** [comment]
Do you think you could add 1-2 sentences to the README.md along with a short example (like all the other methods)?

File: README.md [hunk 1]
```diff
@@ -546,3 +546,22 @@ geolib.convertArea(298678, 'km2'));
 ```
 
 Returns the converted area as number.
+
+
+### `wktToPolygon(wkt)`
+
+Converts the Well-known text (a.k.a WKT) to polygon that Geolib undertands.
```
Comment (mateusppereira) [inline README.md]:
```suggestion
Converts the Well-known text (a.k.a WKT) to polygon that Geolib understands.
```

**Pull request merged**